src/devices/Camera/Camera/Camera.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultSampleTfms)</TargetFramework>
```

Comment:
I think this should be `$(DefaultBindingTfms)`

Reply:
Thank you for your help @krwq. Now everything is ok.